charts/zabbix/values.yaml

```diff
@@ -2,6 +2,18 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
+# Global configurations
+global:
+  # Reference to one or more secrets to be used when pulling images.
+  #
+  # For example:
+  #  imagePullSecrets:
+  #    - name: "image-pull-secret"
+  imagePullSecrets: []
+
+  # Labels to apply to all resources.
```

Comment:
Hi @syphernl!

Can you change this line?

`# -- Labels to apply to all resources.`